src/test/java/net/snowflake/client/annotations/RunOnWindowsOrMac.java

```diff
@@ -0,0 +1,13 @@
+package net.snowflake.client.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@EnabledOnOs({OS.WINDOWS, OS.MAC})
```

Comment:
As in the LinuxOrMac case

Reply:
I mean there that we can annotated the annotation with both `RunOnWindows` and `RunOnMac` instead of redefining the condition as a merge - if it is not possible we can leave it as it is

Reply:
I get you point, but this unfortunately seems to be impossible, at least via standard means. I might try and find a way to make it work later, but I don't think it's worth the effort for now.